moveit_ros/planning/trajectory_execution_manager/src/trajectory_execution_manager.cpp

```diff
@@ -129,6 +129,25 @@ void TrajectoryExecutionManager::initialize()
   else
     allowed_goal_duration_margin_ = DEFAULT_CONTROLLER_GOAL_DURATION_MARGIN;
 
+  XmlRpc::XmlRpcValue controller_list;
```

Comment:
add comment above saying what this section does

also lets just move this into a private function ``loadControllerParams()``